backstop/starrylight.js

```diff
@@ -0,0 +1,9 @@
+var config = require('./core.js');
+
+config.id = 'starrylight';
+config.paths.bitmaps_reference = "then/starrylight";
```

Comment:
Could maybe extract all these `config.paths.` things out into a function in `core.js` (or such) that accepts `layout` and generates them, so here we just have `setConfig(config, 'starrylight');`

Reply:
Possibly but I didn't want to do _more_ javascript things when it was easy enough to do this.

Reply:
That's fair ­– if we ever get around to editing this list, we should clean it up then, then, since the duplication may make it hard to keep consistent.